tests/test_validators.py

```diff
@@ -316,14 +316,26 @@ def test_slug():
     assert iv.check('a-b') == 'a-b'
     assert iv.check('a_b') == 'a_b'
 
-    with pytest.raises(t.DataError):
-        iv.check('_')
-    with pytest.raises(t.DataError):
-        iv.check('')
+    # updates: '_' and empty string are allowed
+    # with pytest.raises(t.DataError):
+    #     iv.check('_')
+    # with pytest.raises(t.DataError):
+    #     iv.check('')
```

Comment:
In this case, just inverse the test code instead of commenting them out.

Reply:
Well noted with thanks!